astropy/utils/iers/tests/test_iers.py

```diff
@@ -346,14 +345,18 @@ def test_IERS_B_parameters_loading_into_IERS_Auto():
 
 
 # Issue with FTP, rework test into previous one when it's fixed
-@pytest.mark.xfail('TRAVIS')
```

Comment:
How about just `skipif(TRAVIS, 'Flaky on travis CI')`? Seems to be no point even running the test if we cannot rely on it!

Reply:
I think we debated about this before and that's why we have it fail on XPASS, but the failing on XPASS makes me sad.

Yeah, good point about not running it on Travis CI at all... 🤔

Reply:
I made the change as you requested, @mhvk .